generators/kubernetes/templates/_service.yml

```diff
@@ -20,11 +20,19 @@ apiVersion: v1
 kind: Service
```

Comment:
I'd rather say split the other specs into separate files. When you change and apply the files, you sometimes don't want to apply all specs from a file, if you just update the deployment. It also stops of overwriting a service, changed by other, when updating a deployment.

Should I split the other files for consistency, too?